targets/targets.json

```diff
@@ -4061,6 +4105,25 @@
             "network-default-interface-type": "WIFI"
         }
     },
+    "SDT8195B": {
+        "core": "Cortex-M3",
```

Comment:
Why this targetr does not reuse realtek  - I can see new files here (HAL implemention) - arent these the same as for realtek?

Reply:
If I just do `"inherits": ["REALTEK_RTL8195AM"]`, I get pin name declaration errors. We see the same error for all the boards, except for SDT64B.

Reply:
I see the problem here

@ARMmbed/team-realtek This target is a platform and can't be used as base (MCU_RTL..... would be needed here?). Please review

Reply:
@0xc0170 There is another issue with K64F. The folder that contains EMAC for K64F is also "TARGET_K64F" (https://github.com/ARMmbed/mbed-os/tree/master/features/netsocket/emac-drivers/TARGET_Freescale_EMAC/TARGET_K64F). This can be pretty confusing to users, because if you want to use the ethernet driver, you need to add `"extra_labels_add": ["K64F"]`.

Target "K64F" should have added `"extra_labels_add": ["K64F"]`, but they didn't have to, because their target name also "happens" to be the same.

Reply:
Is that EMAC implementation generic for any K64F or targetting just frdm-k64f board? If generic, folder should be named TARGET_MCU_K64F (no addition should be needed there, but MCU_K64F is not yet a target but at least there are folders that contain MCU files). Can you create an issue for this? This should be reviewed